build.sbt

```diff
@@ -4,7 +4,7 @@ val gatlingVersion = "2.3.0"
 
 scalacOptions += "-target:jvm-1.8"
 
-libraryDependencies += "com.datastax.dse"             %  "dse-java-driver-core"          % "2.3.0"
+libraryDependencies += "com.datastax.oss"             %  "java-driver-core"              % "4.4.0"
```

Comment:
When you do the upgrade, I think you need to change `DseSession` to `CqlSession` (similar for other deprecated classes)

Reply:
Agreed.  I happened across this when trying to use the plugin with some of the dse-benchmarks; they use DseSession explicitly and the type mismatch was causing trouble.  I've removed all references to DseSession in a pair of recent commits.